packages/design-system/src/styles/components/_Choice.scss

```diff
@@ -123,7 +135,6 @@ $ds-c-inset-border-width: $spacer-half;
 
     &::before {
       background-color: rgba($color-muted-inverse, 0.15);
-      box-shadow: 0 0 0 1px $color-muted-inverse;
```

Comment:
What does this change do?

Reply:
Oh I see, its for the inverse styling.

Reply:
When box-shadow was there for inverse choices only it gave the appearance of a wider border. Take a look a the live documentation site https://design.cms.gov/components/choice/ 

I opted to remove this so the borders can be applied consistently to inverse and non-inverse disabled examples.